src/Context/CountriesDataContext.tsx

```diff
@@ -34,6 +38,7 @@ export const useCountriesContext = () => {
 };
 
 export const CountriesDataContext = ({ children }: { children: ReactNode }) => {
+	const [selectOption, setSelectOption] = useState("Filter by Region");
```

Comment:
[selectedOption, setSelected] - change the name :)

Reply:
done.